### PR TITLE
[IMP] hr_attendance_validation: Ignoring some leave types

### PR DESCRIPTION
**Pull request by petrus-v**
we are using hr.leaves to manage allowed remote days but we don't want to remove thoses times in attendance validation

### CODE REVIEW

File: hr_attendance_validation/tests/test_hr_attendance_validation.py
Comment (petrus-v) [inline hr_attendance_validation/tests/test_hr_attendance_validation.py:109]:
```suggestion
```

**Review by FrancoMaxime — APPROVED** [review]
lgtm: code review

**Comment by rafaelbn** [comment]
Hello @OCA/human-resources-maintainers , please could you reopen this PR?

Thank you! 🙏🏼 
Regards 😄 ❤️

**Comment by rafaelbn** [comment]
Hello @petrus-v ,

I you are the author of this module, you can add your self as maintainter so you can merge your self

Do you know about it?

Regards ✋🏼

**Comment by rafaelbn** [comment]
Sorry for noise, @nimarosa could you reopen and merge?

**Review by nimarosa — APPROVED** [review]

**Comment by nimarosa** [comment]
/ocabot rebase

@rafaelbn lets rebase first so we check the tests passes. Then we can merge.

**Comment by nimarosa** [comment]
/ocabot merge minor 

@petrus-v @rafaelbn please after merge cherry peak this PR to other versions so we maintain same functionality in all versions.

**Comment by petrus-v** [comment]
Thanks @rafaelbn to make this merge possible !

Yes, sometimes I try to put myself as maintainer but I've never use the `ocabot` command yet ! I've to read a bit more about it to feel confident to merge stuff !

**Pull request merged**
